golang/template/cluster-desc.sample.yaml

```diff
@@ -22,12 +25,18 @@ coreos_channel: "stable"
 coreos_version: "1122.2.0"
 
 # centos version
-download_kernel: n
-centos_version: "7.3.1611"
+download_kernel: y
+# mirror_site, please check that $mirror_site/$centos_version/(isos|os) exist.
+# sunch as:
+#   http://mirrors.163.com/centos
+#   http://vault.centos.org/
+mirror_site: http://vault.centos.org
+# please note this version number, it's important.
```

Comment:
`# please note this version number, it's important.` => `# Configure the centos version for isos and rpms`